vehicle/raw_vehicle_cmd_converter/include/raw_vehicle_cmd_converter/accel_map.h

```diff
@@ -22,22 +22,24 @@
 #include <string>
 #include <vector>
 
-#include <ros/ros.h>
+#include <rclcpp/rclcpp.hpp>
```

Comment:
FYI In general, the order of includes should be reversed, from the project files to external dependencies to std includes coming last; see e.g. https://blog.knatten.org/2010/07/01/the-order-of-include-directives-matter/

But we don't want to do such changes in this porting. So nothing to do here but keep in this in mind for all your new files. And perhaps refactoring this later

Reply:
fixed.